step.yml

```diff
@@ -61,19 +61,24 @@ inputs:
       title: "Send To emails"
       description: |
         Comma separated email address list.
-      summary: ""
+      summary: "
       is_required: true
-  - subject: "Build finished"
+  - subject: "$BITRISE_APP_TITLE Build $BITRISE_BUILD_NUMBER finished"
     opts:
       title: "Email Subject"
       description: ""
       summary: ""
       is_required: true
-  - message: "Build succed on Bitrise.io"
+  - message: "$BITRISE_APP_TITLE Build $BITRISE_BUILD_NUMBER succeeded on Bitrise.io"
     opts:
       title: "Email Message"
       description: |
-        The message can be HTML or plain text.
+        Build succeeded!
```

Comment:
please don't change the description - it's not the default value of the step but the description of the input, which is presented on the UI ;)